en/assembly/quick_start_cuav_v5_nano.md

```diff
@@ -14,7 +14,7 @@ We'll go through each of these in detail in the following sections.
 
 | Main interface | Function |
 | :--- | :--- |
-| Power | Connect Power module;  power input & AD voltage  and current detection. |
+| Power | Connect Power module; Power input and AD voltage and current detection. |
```

Comment:
Connect Power module; Provides Power and ANALOG voltage and current measurements.